tools/trap/ramdumpParser.py

```diff
@@ -1058,7 +1071,7 @@ def main():
 
 		# Reading the Current Task from g_readytorun list, usually head is the current
 		# Since its a pointer, first am getting pointer address and then reading the
-		# contents of that Pointer to get the proper task address
+		# Contents of that Pointer to get the proper task address
```

Comment:
This again should be decapitalized, as it is a continuation of the sentence in the above line.